SYCL/ESIMD/spec_const/spec_const_bool.cpp

```diff
@@ -9,6 +9,7 @@
 // RUN: %clangxx -fsycl -I%S/.. %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // UNSUPPORTED: cuda || hip
+// XFAIL: esimd_emulator
```

Comment:
TODO?

Reply:
Like I showed in messenger, online_compiler check fails for all spec* tests - even if it is not required. To whom should I ask about this issue? (Someone in SYCL_RT?)